uk.co.compucorp.civicrm.hrleaveandabsences/settings/leaveandabsence.setting.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+  return [
+    'relationship_allowed_to_approve_leave' => [
+      'group_name' => 'Leave and Absence General Settings',
+      'group' => 'leaveandabsence',
```

Comment:
To be honest, I don't know the difference. I only followed the convention I found in Core.
I'll change the name though